frontend/apps/crates/entry/user/src/register/pages/step_2/dom.rs

```diff
@@ -24,6 +25,10 @@ pub struct Step2Page {}
 
 impl Step2Page {
     pub fn render(step: Mutable<Step>, step_1: Step1Data) -> Dom {
+        static LINK_COLOR: Lazy<String> = Lazy::new(|| class! {
+            .style("color", "var(--main-blue)")
+        });
+        
```

Comment:
Hey @MendyBerger, what's your opinion of using styles here?

Moving the anchor styles over to the Web Component side for these anchor tags feels very specific for logic that lives in the `input-checkbox` component. I moved the other styles into `input-checkbox` but should I leave the anchor colors in Rust?

Reply:
Hmm..., I think we should chat about this face to face, gimme a ping when you're available